lib/install/validate/network.go

```diff
@@ -347,3 +373,78 @@ func (v *Validator) checkVDSMembership(ctx context.Context, network types.Manage
 	}
 	return nil
 }
+
+// suggestNetwork suggests all networks (only shows DPG on vC)
```

Comment:
So far, we still allow standard network used in VC, if it's not used for bridge network and container network.
So maybe your suggestion code can add one more flag, for if standard network is expected.